Patches/Star Wars - Droids/Patch_SWDroids_Core.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Star Wars - Droids Core (Continued)</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			
+			
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[
```

Comment:
It seems like a few of those various attacks should have different damages; Cut for some of the saws, Burn for the welder, Stab for the injector, Blunt for the arm, etc.

Reply:
I think these are placeholders. When installed on the droids it should use their melee attacks instead.